spec/order_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../lib/order'
+
+describe Order do
+  
+  subject(:order) { described_class.new } # allows the use of 'dish' instead of 'subject' for clarity  
+  
+  describe "#show_menu" do
```

Comment:
Really like that you've broken up each 'function' into separate 'describe' blocks, makes it much easier to read!